molecule/test-local/converge.yml

```diff
@@ -106,12 +110,18 @@
         src: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') }}/deploy/non-olm/latest/operator.yml"
         dest: "{{ tmp_dir.path }}/operator.yml"
 
-    - name: Updat Deployment
+    - name: Update Deployment
       lineinfile:
         path: "{{ tmp_dir.path }}/operator.yml"
         regexp: 'apiVersion: apps/v1beta1'
         line: 'apiVersion: apps/v1'
 
+    - name: Update Pull Policy so we use the image we just finished building
+      lineinfile:
+        path: "{{ tmp_dir.path }}/operator.yml"
+        regexp: '        imagePullPolicy: Always'
+        line: '        imagePullPolicy: Never'
```

Comment:
This will solve the problem of test not really running against PRs. The image gets built and tagged by molecule above, but since the imagePullPolicy is Always in the operator.yml it goes out and pulls the latest copy from quay instead of using the local copy. We change that to Never here and now it we're testing our changes (which can be evidenced by the lack of k8s_facts deprecation warnings int he travis run, as well as pushing someting like `-fail: msg="test"` in the operator role and seeing tests fail.